### PR TITLE
Font Awesome 5 upgrade

### PR DESCRIPTION
**Pull request by stevemulligan**
Removes Font Awesome 4 and adds Font Awesome 5 css and font source files.  Upgrades icons on the home page and install page.

The reason for Font Awesome 5 is they added icons for Discord, Telegram and Bitcoin which will be nice for adding social media links to the front page.

### CODE REVIEW

File: _sass/_syntax.scss [hunk 1]
```diff
@@ -21,7 +21,8 @@ figure.highlight {
     padding: 0.5em;
     background-color: $lighter-gray;
     content: "\f121";
-    font-family: "fontawesome" !important;
+    font-family: "Font Awesome 5 Free" !important;
+		font-weight: 900;
```
Comment (ellaismer) [inline _sass/_syntax.scss]: Should use 4 whitespace this line.

**Review by ellaismer — COMMENTED** [review]
Icons do not display for me when running `jekyll serve`. Any hint what's happening?

**Comment by stevemulligan** [comment]
In _config.yml I had to set `baseurl                  : ""`  So that the css and other assets would be pulled from my local environment.

**Comment by ellaismer** [comment]
@stevemulligan Oh yeah I missed that. Or another way is to just use the `_config.dev.yml` config.

```
bundle exec jekyll serve --config _config.yml,_config.dev.yml
```

**Review by ellaismer — APPROVED** [review]

**Pull request merged**